docs/book/src/getting-started/testdata/project/internal/controller/memcached_controller.go

```diff
@@ -409,11 +409,8 @@ func labelsForMemcached(name string) map[string]string {
 	if err == nil {
 		imageTag = strings.Split(image, ":")[1]
 	}
-	return map[string]string{"app.kubernetes.io/name": "Memcached",
-		"app.kubernetes.io/instance":   name,
-		"app.kubernetes.io/version":    imageTag,
-		"app.kubernetes.io/part-of":    "project",
-		"app.kubernetes.io/created-by": "controller-manager",
+	return map[string]string{"app.kubernetes.io/name": "project",
+		"app.kubernetes.io/version": imageTag,
```

Comment:
We should have `app.kubernetes.io/managed-by: {{ KindController}}`

Reply:
sorry for missing this